src/control/indexOperationsBuilder.ts

```diff
@@ -0,0 +1,27 @@
+import {
```

Comment:
This seemed a bit silly to unit test, and the only thing we'd really care about checking is the proper `controllerPath` being set. Since the internals of the OpenAPI classes are protected, that's not necessarily easy unless we want to use a `ts-ignore`.